src/Searcher.tsx

```diff
@@ -42,7 +38,7 @@ export class Searcher {
         this.displayedCategories = categories;
     }
 
-    generateStructuredItems(): JSX.Element[] {
+    generateStructuredItems(showExpandableToolTip: boolean): JSX.Element[] {
```

Comment:
I'm thinking of not calling it a *tool tip* because no matter who looks at it, they will have the conventional floating tooltip window in mind. If we can call it *ItemDescription*, then this parameter can be named `displayItemDescription`.